src/app/components/Figure/index.jsx

```diff
@@ -1,15 +1,28 @@
 import React from 'react';
 import { string } from 'prop-types';
+import styled from 'styled-components';
 import Caption from './Caption';
 
 const renderCaption = caption =>
   caption ? <Caption>{caption}</Caption> : null;
 
+const StyledFigure = styled.figure`
+  padding-bottom: 16px;
+  width: 100%;
+`;
+
+const Image = styled.img.attrs({
+  src: ({ src }) => src,
+  alt: ({ alt }) => alt,
```

Comment:
You don't need these two lines for `src` and `alt`.

Reply:
Ok